### ISSUE TITLE
[FR] Add support for regex patterns in exclude 

### ISSUE THREAD

**Issue by MarArMar** [cboxdoerfer/fsearch#563]
**Is your feature request related to a problem? Please describe.**
I want to exclude **/node_modules/** and **/.venv/** from my searches (installed libraries in software projects)

**Describe the solution you'd like**
In the exclude, accept regex patterns

**Describe alternatives you've considered**
Listing all my modules folders but there are many

**Comment by cboxdoerfer** [comment]
Yes, more flexible exclude patterns are planned. I've also been experimenting with a better UI for that, where for each exclude entry you can specify its scope (file, folder, name, full path) and the pattern type (regex, wildcard, literal).

**Comment by cboxdoerfer** [comment]
This has been implemented and will be part of the 0.3 release. You can try it out on master or the beta flatpak.

**Comment by MarArMar** [comment]
❤️ tysm !!
